.vib/redis/sentinel/goss/goss.yaml

```
@@ -1,16 +1,16 @@
 command:
   redis-sentinel-auth:
-    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h redis -p 26379
+    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h redis -p 80
```

I'd change the host rather than the port, which I believe is less likely to be changed. This relies on the LoadBalancer name, which in turn relies on the release name.
```suggestion
    exec: REDISCLI_AUTH="$REDIS_PASSWORD" redis-cli -h localhost -p 26379
```